docs/navigation-actions.md

```diff
@@ -42,7 +42,7 @@ const navigateAction = NavigationActions.navigate({
 this.props.navigation.dispatch(navigateAction);
 ```
 
-### Back
+### back
 
 Go back to previous screen and close current screen. `back` action creator takes in one optional parameter:
```

Comment:
Nit: there is `back` action in this paragraph that should be capitalized

Reply:
they should all be camelcase otherwise its confusing to the user